logger.go

```diff
@@ -360,8 +360,15 @@ var timeOffset, timeZone = func() (int64, string) {
 	return int64(n), s
 }()
 
+var globalLevel = new(Level)
+
+
+func SetGlobalLevel(level Level) {
```

Comment:
I `SetGlobalLevel` name is not strong enough to indicate it will "override" the logger's level.
Maybe `SetGlobalLevelForAllLoggers` is stronger but a bit long.

Do you have a better idea?

Reply:
Agree with you that `SetGlobalLevel` is not a good name. Also have no idea about a better name..

Reply:
`SetGlobalLevelForAllLoggers` is clear and have some 'warn' meaning, at lease better than `SetGlobalLevel`..